scripts/apps/authoring-react/authoring-react.scss

```diff
@@ -20,7 +20,6 @@
     justify-content: space-between;
     gap: 16px;
     align-items: center;
-    padding-left: 16px;
```

Comment:
if you're removing padding here, you need to [update](https://github.com/superdesk/superdesk-client-core/assets/2076953/c88ab369-c057-4c9a-8ceb-3e9758109df6) `ITopBarWidget['group']`  (and update usages)  to be:

```typescript

interface ITopBarWidgetGroupSpacing {
    noSpacingBefore?: boolean;
    noSpacingAfter?: boolean;
}

interface ITopBarWidgetGroupStart extends ITopBarWidgetGroupSpacing {
    position: 'start';
}

interface ITopBarWidgetGroupMiddle extends ITopBarWidgetGroupSpacing {
    position: 'middle';
}

interface ITopBarWidgetGroupEnd extends ITopBarWidgetGroupSpacing {
    position: 'end';
}

export interface ITopBarWidget<T> {
    component: React.ComponentType<{entity: T}>;
    availableOffline: boolean;
    priority: IDisplayPriority;
    group: ITopBarWidgetGroupStart | ITopBarWidgetGroupMiddle | ITopBarWidgetGroupEnd;
    keyBindings?: IKeyBindings;
}
```